app/controllers/users_controller.rb

```diff
@@ -1,3 +1,48 @@
 class UsersController < ApplicationController
-  def new; end
+  def new
+    @user = User.new
+  end
+
+  def show
+    @user = User.find(params[:id])
+    return if @user
+
+    flash[:warning] = t(".error.not_found").to_s
+    redirect_to root_path
+    # debugger
+  end
```

Comment:
```suggestion
  def show; end
```
Đưa các code bên trong ra thành hàm load_user, và call trong before_action